android/assets/jsons/translations/Spanish.properties

```diff
@@ -250,10 +248,8 @@ Our items = Nuestros productos
 Our trade offer = Nuestra oferta de comercio
 [otherCiv]'s trade offer = Oferta de [otherCiv]
 [otherCiv]'s items = Productos de [otherCiv]
- # Requires translation!
-+[amount] untradable copy = 
- # Requires translation!
-+[amount] untradable copies = 
++[amount] untradable copy = +[amount] copia incomercial
++[amount] untradable copies = +[amount] copias no-comerciables
```

Comment:
Dos versiones distinctas por lo mismo?

Reply:
One is in plural and the other in singular...